README.md

```diff
@@ -89,6 +89,25 @@ Follow these steps to install the `chasten` program:
 - Type `pipx list` and confirm that Chasten is installed
 - Type `chasten --help` to learn how to use the tool
 
+## 🐋 Docker
+
+There is also the option to use [Docker](https://www.docker.com/) to interact with `chasten`
```

Comment:
Instead of the words "interact with", I suggest that we write "use". See what I mean?